index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;500&family=Quicksand:wght@300;500&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/d1333a485b.js" crossorigin="anonymous"></script>
+
+<body>
+
+  <header class="header-container">
+    <img src="images/figures-small1.png" alt="header figures">
```

Comment:
- [ ] [OPTIONAL] Just as the previous reviewer has mentioned, it will be proper if you make the image a background image since it is a decorative image (_an image that is non-essential to understanding the content and purpose of a page_). And decorative images are better displayed as background images. What do you think? 😊😊